template.yml

```diff
@@ -262,6 +262,82 @@ Resources:
       Environment:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/change_runtime_directory
+          OUTPUT_BUCKET_NAME: !Ref OutputBucket
```

Comment:
not sure if this is the best way to get these parameters over to the lambda, if there's a better pattern let me know!

Reply:
I think this is the standard mechanism to pass configuration settings to a Lambda :)